compiler/codegen/type.cpp

```diff
@@ -350,8 +350,14 @@ void AggregateType::codegenDef() {
 
         llvm::StructType * st;
         // handle an empty union.
-        if( largestType ) st = llvm::StructType::get(largestType, NULL);
-        else st = llvm::StructType::get(info->module->getContext());
+        if( largestType ) {
+          st = llvm::StructType::get(largestType
+#if HAVE_LLVM_VER < 50
+                                     , NULL
+#endif
+                                     );
+        } else
+          st = llvm::StructType::get(info->module->getContext());
```

Comment:
This is a trivial/style thing, but I always use curly braces on an else block if I used them on an if block...

Reply:
Sounds reasonable.  I will push a fix to make the curly braces the same for the else block.